build/target/etc/redis.conf

```diff
@@ -0,0 +1,1316 @@
+# Redis configuration file example.
```

Comment:
Is it needed to have the full redis sample config?

Reply:
That's how we had it in LiskHQ/lisk-scripts (default config with a few changes); the advantage of having the full file is that it can be easily `diff`ed against a newer default configuration.